include/sched.h

```diff
@@ -0,0 +1,70 @@
+#ifndef SCHED_H
+#define SCHED_H
+
+#include "ultra64.h"
+#include "irqmgr.h"
+
+#define OS_SC_NEEDS_RDP     0x0001  // Task uses the RDP
+#define OS_SC_NEEDS_RSP     0x0002  // Task uses the RSP
+#define OS_SC_DRAM_DLIST    0x0004  // RDP commands generated in the RSP pass through DRAM
+#define OS_SC_PARALLEL_TASK 0x0010  // Unimplemented
+#define OS_SC_LAST_TASK     0x0020  // Unimplemented
```

Comment:
`LAST_TASK` is actually used in `graph.c`, it just does nothing. I'd like to keep `PARALLEL_TASK` to explain why there'd otherwise be a weird gap in the flags.